lib/main.dart

```diff
@@ -31,32 +30,33 @@ class _MyWidgetState extends State<MyWidget> {
   Widget build(BuildContext context) {
     return Scaffold(
       body: Center(
```

Comment:
<table data-meta="generated_by_danger-pr">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kolbe-ryo/count_habits/blob/fcc666675c96c51a87a3d79e22d0c947e8ed3891/lib/main.dart#L32"></span>Use 'const' with the constructor to improve performance</td>
    </tr>
  </tbody>
</table>

Reply:
[9e01d6b](https://github.com/kolbe-ryo/count_habits/pull/7/commits/9e01d6b548b66e2008f7dfdcc1a8ac931512066f)